packages/api/helper/mailer.js

```diff
@@ -38,15 +41,33 @@ function sendDeclineEmail(nomination) {
     template: 'decline',
     message: {
       from: 'Bill <bill@keepswimmingfoundation.org>',
-      // to: 'sophia@thedifferenceengine.co'
       to: nomination.providerEmailAddress,
     },
     locals: {
       name: nomination.providerName,
       patientName: nomination.patientName,
       appUrl: process.env.APP_URL,
     }
-  }.catch((err) => console.log(err))).then(() => console.log('email has been sent!'));
+  }).catch((err) => console.log(err)).then(() => console.log('email has been sent!'));
+}
+
+function sendSurveyEmail(nomination) {
+  email.send({
+    template: 'survey',
+    attachments: './survey/header.jpg',
+    message: {
+      from: 'Bill <bill@keepswimmingfoundation.org>',
```

Comment:
Looks like the string for Bill's email is being used multiple times now. In these situations it helps to save the string off in a variable like `BillEmail` or something so then you can just reference `BillEmail` multiple times and not risk mistyping the string if you need to use it again.

Reply:
I used adminEmail just incase.